src/main/java/org/mozilla/jss/util/jssutil.c

```diff
@@ -926,6 +926,271 @@ JSS_clearPtrFromProxy(JNIEnv *env, jobject nativeProxy)
     }
 }
 
+/* V2 refers to PKCS #5 V2 here. If a PKCS #5 v1 or PKCS #12 pbe is passed
```

Comment:
Would be a good idea to identify which portion of the code has been copied from nss to 1. save time for the reviewers, and 2. for future